app/models/cms/file.rb

```diff
@@ -16,7 +16,8 @@ class CMS::File < ApplicationRecord
   has_attached_file :attachment,
                     url: Rails.application.config.cms_files_path,
                     s3_url_options: ->(file) { file.s3_url_options },
-                    validate_media_type: false
+                    validate_media_type: false,
+                    adapter_options: { hash_digest: Digest::SHA256 }
```

Comment:
For this we need to rehash all attachments which might be an issue for updating existing deployments. See [paperclip docs](https://www.rubydoc.info/gems/paperclip/6.1.0#checksum-fingerprint).

Reply:
Ya agree there.  Looks like it can be fixed with something like:

```ruby
CLASS='CMS::File' ATTACHMENT=attachment rake paperclip:refresh:fingerprints
```

so that might be something we need to add to the docs.